test/unit/puppetlabs/services/request_handler/request_handler_core_test.clj

```diff
@@ -97,7 +99,7 @@
                              :content-type       "text/plain"
                              :character-encoding "UTF-16"
                              :params             {:bogus ""}})]
-      (is (= {} (:params wrapped-request))
+      (is (= {:bogus ""} (:params wrapped-request))
```

Comment:
'tis a bit weird to have `:bogus` as the expected value now.  Perhaps change that key to something else like `:notbogus`.